packages/hash/frontend/src/components/hooks/blockProtocolFunctions/knowledge/useBlockProtocolGetEntity.ts

```diff
@@ -95,6 +95,10 @@ export const useBlockProtocolGetEntity = (): {
           query: getEntityTypeRootedSubgraphQuery,
```

Comment:
This should also be removed. It's redefining the query that's already given at instantiation.

Reply:
Will do that when I touch the entity endpoints. I had removed it locally and then changed my mind. I'd like to keep a separation of concerns